libs/components/ag-grid/package.json

```diff
@@ -35,8 +35,8 @@
     "@skyux/lookup": "0.0.0-PLACEHOLDER",
     "@skyux/popovers": "0.0.0-PLACEHOLDER",
     "@skyux/theme": "0.0.0-PLACEHOLDER",
-    "ag-grid-angular": "^31.1.1",
-    "ag-grid-community": "^31.1.1"
+    "ag-grid-angular": "^31.2.0",
+    "ag-grid-community": "^31.2.0"
```

Comment:
Should these versions match what's in the `packageGroup`? `~31.2.0`

Reply:
Good call.